env.sh

```diff
@@ -0,0 +1,2 @@
+export MONGODB_URI="mongodb://fortnite:pass123@ds231242.mlab.com:31242/fortnite-lotto";
```

Comment:
- Environment variables shouldn't be tracked on git nor should they be exported otherwise others can see your username and pw for the db connection
- They should be contained in the `.env` file, which is ignored by the `.gitignore` file